src/core/index.ts

```diff
@@ -13,6 +13,7 @@ import { dismissAllApprovals } from "./dismiss_approval";
 import { getChangedFiles } from "./get_files";
 import { getOwnershipRules } from "../utils/config_parser";
 import { getUserInfo } from "./get_info";
+import { isUserBlacklisted } from "../utils/matchers";
```

Comment:
Codacy found an issue: [Multiple imports from '../utils/matchers' can be combined into one.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6336596)